### DISCUSSION TITLE
Очистка лога

### DISCUSSION THREAD

**Discussion by bear-gha** [Anonym-tsk/nfqws-keenetic#104] [Идеи и предложения]
Приветствую, просьба сделать кнопку очистки лога в web пакете. Как вариант - там же где появляется кнопка Save.

`cat /dev/null > /opt/var/log/nfqws.log`

**Comment by EasyKill33** [comment]
А так же может быть есть возможность добавить кнопку обновления версии в интерфейс?

**Reply by Anonym-tsk** [reply]
Это уже сделано

**Comment by Anonym-tsk** [comment]
А зачем вам чистить лог, если не секрет?

**Reply by bear-gha** [reply]
Не люблю рыться в длинных логах, да и если "забить", то его "разнесет". Плюс когда идет какая-либо отладка, проще чистить и наблюдать.

**Comment by bear-gha** [comment]
Обновился, огонь, спасибо 👍 .
